src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/ApplicationsTransformer.kt

```diff
@@ -190,4 +191,15 @@ class ApplicationsTransformer(
     AssessmentDecision.REJECTED -> uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.AssessmentDecision.rejected
     null -> null
   }
+
+  fun transformTimelineToApi(events: List<JpaDomainEventSummary>): List<APIDomainEventSummary> {
+    return events.map {
+      it
+      APIDomainEventSummary(
+        type = it.type.toString(),
```

Comment:
Similarly, I think we might want to have a new TimelineEvent type here - again, they don't 
_quite_ map 1:1 with the domain events we have, and I think it gives us more flexbility to seperate the domain events from the timelines